### PR TITLE
BloodHound: add suppport for arm64 builds

### PR DESCRIPTION
**Pull request by vashirov**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

File: Casks/bloodhound.rb [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+  app "BloodHound-darwin-#{arch}/BloodHound.app"
+
+  caveats <<~EOS
+    According to https://github.com/BloodHoundAD/BloodHound/issues/504
+    this app will not work with quarantine attributes.
+    To use this cask, install it with:
+
+      brew install --cask --no-quarantine #{token}
+
+    WARNING: quarantining is a macOS security feature and you disable it at your own risk!
```
Comment (SMillerDev) [inline Casks/bloodhound.rb:26]: Definitly don't want to recommend this. 
```suggestion
```

Comment (vashirov) [reply Casks/bloodhound.rb:26]: For the record, the same wording exists in https://github.com/Homebrew/homebrew-cask/blob/master/Casks/kieler.rb and https://github.com/Homebrew/homebrew-cask/blob/master/Casks/universal-gcode-platform.rb

Comment (SMillerDev) [reply Casks/bloodhound.rb]: Thanks for pointing that out, I've made pull requests to rectify it.

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @vashirov  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**